src/Conversion/ONNXToKrnl/Math/Elementwise.cpp

```diff
@@ -863,6 +863,17 @@ struct ONNXElementwiseUnaryOpLowering : public ConversionPattern {
     auto loc = ONNXLoc<ElementwiseUnaryOp>(op);
     auto X = operands[0];
 
+    // If type is scalar, there need not allocate a buffer. Just call scalar
```

Comment:
If type is scalar, there need not allocate a buffer
-> If the type is a scalar, there is no need to allocate a buffer

Reply:
Done. Thanks!